cmd/ak/cmd/mcp/utils.go

```diff
@@ -0,0 +1,66 @@
+package mcp
+
+import (
+	"fmt"
+
+	"github.com/mark3labs/mcp-go/mcp"
+
+	"go.autokitteh.dev/autokitteh/cmd/ak/common"
+	"go.autokitteh.dev/autokitteh/internal/resolver"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+)
+
+func resolveOrg(org string) (sdktypes.OrgID, error) {
+	if org == "" {
+		return sdktypes.InvalidOrgID, nil
+	}
+
+	ctx, cancel := common.LimitedContext()
+	defer cancel()
+
+	r := resolver.Resolver{Client: common.Client()}
+	oid, err := r.Org(ctx, org)
+	if err != nil {
+		return sdktypes.InvalidOrgID, fmt.Errorf("resolve org: %w", err)
+	}
+
+	return oid, nil
+}
+
+func getOptionalParam[T any](args map[string]any, key string) (*T, error) {
```

Comment:
Why is this and `getMandatoryParam` defined here when they are used only in `tools.go`?

Reply:
merged `utils.go` into `tools.go`.